src/runtime/runtime-debug.cc

```diff
@@ -1188,6 +1188,9 @@ static std::string GetStackLocation(Isolate* isolate) {
     }
     std::vector<FrameSummary> frames;
     CommonFrame::cast(frame)->Summarize(&frames);
+    if (!frames.size()) {
+      continue;
+    }
```

Comment:
This fixes a crash when getting the topmost frame location.

Reply:
Can you link to a crash report that captures this? Or was this only something you saw intermittently in your own testing?

Reply:
@jazzdan hit this while testing but we didn't get crash report data due to https://linear.app/replay/issue/TT-1114/crashes-from-libcpp-aborts-are-not-reported